Test/GroundState/test_groundstate.py

```diff
@@ -98,7 +99,8 @@ def test_vmc_iterator_iterative():
         for name in "Energy", "SigmaX":
             assert name in obs
             e = obs[name]
-            assert hasattr(e, "mean") and hasattr(e, "variance") and hasattr(e, "R")
+            assert hasattr(e, "mean") and hasattr(
+                e, "variance") and hasattr(e, "R")
```

Comment:
This is not the [black](https://github.com/psf/black) code style we [agreed on previously](https://github.com/netket/netket/pull/190), which is also the reason why this PR seems to contain so many changes that are purely formatting. Could you run `black` on all the files you changed? This should eliminate these unneeded changes.

Reply:
We don't have to do this in this PR though. I can also make a separate PR later, once we have the changes for 2.1 finalized.

Reply:
sure I can do that... even though I find that having lines than are too long really hurts readability, so I am not sure black is doing a reasonable job there

Reply:
The line length is something that can be changed. [Black uses 88 as default](https://github.com/psf/black#line-length) specifically, because the developers found that in the code they looked at there were often lines slightly over the 80 character limit.

But it is possible to change that default to 80 or maybe 82, if you prefer. Could be worth a try.